signatures/gemss/gemss/impl/opt.yaml

```diff
@@ -0,0 +1,8 @@
+name: Optimized
+type: optimized
+platform: c
+dependencies:
```

Comment:
this only refers to hardware dependencies, like aes instructions, avx2 and the like